core/src/runtime/syscall.rs

```diff
@@ -127,21 +123,17 @@ impl<'a, 'h> SyscallContext<'a, 'h> {
     self.rt.word(addr)
   }
 
+  pub fn dword(&self, addr: u32) -> u64 {
```

Comment:
meaning of word, dword, qword and so on depends on architecture I think. Here, I used dword for a `u64` because a `u32` is named "word". So double-word = "2 * u32" = u64.